Makefile

```diff
@@ -92,6 +85,16 @@ CONTROLLER_GEN=$(shell which controller-gen)
 endif
 .PHONY: controller-gen
 
-are-you-sure:
-	@echo "Are you sure? [Y/N] " && read ans && [ $${ans:-N} = Y ]
-PHONY: are-you-sure
+quickstart:
```

Comment:
A little easier way to do this is to use `envsubst`. You can put ${AZURE_IDENTITY} in the yaml file and then do `cat quickkstart.yaml | envsubst`. `envsubs` will replace those strings with what is set in your environment.

Reply:
Oh yeah, even if I'd stuck with 'sed' I forgot to clean that up more.